src/System.Windows.Forms/tests/UnitTests/System/Windows/Forms/PropertyGridInternal/PropertyGridView.DropDownHolderTests.cs

```diff
@@ -35,4 +36,80 @@ public void DropDownHolder_CreateAccessibilityObject_creates_DropDownHolderAcces
         AccessibleObject accessibleObject = dropDownHolder.AccessibilityObject;
         Assert.Equal("DropDownHolderAccessibleObject", accessibleObject.GetType().Name);
     }
+
+    [WinFormsFact]
+    public void DropDownHolder_SetDropDownControl_control_notnull()
+    {
+        using PropertyGrid propertyGrid = new PropertyGrid();
```

Comment:
nit: we can simplify the initialization if the type is on the left hand side.
```suggestion
        using PropertyGrid propertyGrid = new();
```
There are a few others below as well.